extension/data/deinflect.dat

```diff
@@ -333,4 +336,34 @@ imperative negative
 り	る	640	24
 れ	る	640	23
 れ	れる	384	24
-ろ	る	384	23
\ No newline at end of file
+ろ	る	384	23
+です	だ	8320	13
+だった	だ	8320	14
+でした	だ	8320	6
+ではない	だ	8320	15
```

Comment:
This hadn't come up before but since this ends in ない that means it doesn't count as `OTHER` anymore and instead counts as an i-adjective. (which has type mask 0x0004). According to my quick calculation that means this type mask should be 8196. I think if you make this change for every mapping that ends in ない then you don't need to add the inflections of ない directly.

```suggestion
ではない	だ	8196	15
```